docs/release/trg-9/trg-9-01.md

```diff
@@ -0,0 +1,75 @@
+---
+title: TRG 9.01 - UX consistency / Styleguide for UI
+sidebar_position: 1
+---
+
+| Status | Created       | Post-History                                             |
+|--------|---------------|----------------------------------------------------------|
+| Active | 22-Jul-2024   | Finalize TRG                                             |
+| Draft  | 10-Jan-2024   | Transferred from Release Acceptance Criteria into TRG    |
+| Draft  | 25-April-2024 | Added new styleguide criteria - initial version          |
+
+## Introduction
+
+The purpose of this guideline is to ensure that business applications with user interfaces adhere to the established UI/UX styleguide. It outlines all the relevant criteria that must be checked when releasing a new product version. A crosscheck with every pull request is not just optional, but it is a "MUST" criteria to ensure the quality and consistency of the user interfaces of each product/component. This step is crucial in maintaining the overall standard and enhancing as well as standardizing the user experience.
```

Comment:
Styleguide mentioned but not linked. This TRG explains itself as a guideline and refers to a styleguide not linked. Maybe I'm a littele picky here, but it's hard to understand for outsiders.

I propose to frame the whole TRG as the UX consistency guidelines to make it consistent.

Reply:
thanks for your feedback, I tried to integrate it